io/plugins/eu.esdihumboldt.hale.io.jdbc.ui/src/eu/esdihumboldt/hale/io/jdbc/ui/SchemasRetrievalPage.java

```diff
@@ -148,6 +148,24 @@ protected void onShowPage(boolean firstShow) {
 				return;
 			}
 
+			Connection conn = getConnection();
+
+			if (conn == null) {
+				disposeControl();
+				// message
+				Label label = new Label(page, SWT.WRAP);
+				label.setText(
+						"Could not established connection with database. Please check the settings.");
```

Comment:
It would be more helpful to tell the user what went wrong, by also including the message in the thrown SQLException (line 195), like this: "Could not establish connection with database: [Error Message]"